src/ProjectNeon/Assets/Data/Templates/TurnAI.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using UnityEngine;
 
-public class TurnAI : ScriptableObject
+public abstract class TurnAI : ScriptableObject
 {
-
+    public abstract PlayedCard play();
```

Comment:
All public members must be pascal-cased.

Reply:
@SilasReinagel Fixed